README.md

```diff
@@ -24,16 +24,23 @@ This document describes two ways of installing Plumber: as an IIS website or run
 
 ### Using IIS
 
-You can download the latest compiled version of Plumber-sc from [releases](https://github.com/ewerkman/plumber-sc/releases). The file `release.zip` contains a pre-compiled version of Plumber that you can install as an IIS site. 
+You can download the latest source code of Plumber-sc from [releases](https://github.com/ewerkman/plumber-sc/releases).
 
-Make sure you have installed the IIS Rewrite module (https://www.iis.net/downloads/microsoft/url-rewrite)
+Make sure you have:
 
-To use IIS to host Plumber-sc:
+- Node.js
+- IIS Rewrite module (https://www.iis.net/downloads/microsoft/url-rewrite)
 
-1. In IIS Manager, create a new website called `plumber-sc`, use port `8080` and set the folder to host the application.
-2. Copy the contents of `release.zip` to the folder you specified in the previous step.
-3. Configure Plumber, Sitecore Identity Server and the commerce engine. Instructions are in the following paragraphs.
+To use IIS to host Plumber-sc:
 
+1. Copy the `plumber-sc-[version]` folder from the release ZIP file into `C:\inetpub\wwwroot`.
```

Comment:
Hi @jflheureux, first of all thanks for this update and sorry for not responding sooner.

I was a little puzzled by some of your suggestions: you should be able to install Plumber from the release.zip package. If not let me know so I can look into this. 
Did you just wanted to provide documentation on how to install Plumber from source?

Reply:
Now I understand!
In the Readme file of https://github.com/plumber-sc/plumber-sc, the "Releases" link navigates to your private fork of the project https://github.com/ewerkman/plumber-sc/releases and there are no releases in your fork.

By manually going in the releases tab of the plumber-sc repo, I can see the releases ZIP files.

You can discard my pull request and fix the link to navigate to the right release page.

Reply:
That is strange because I can see them even when logged out. So if you go here: https://github.com/plumber-sc/plumber-sc/releases you don't see anything?

Reply:
Ah, that explains a lot. Thanks for helping fix that!